tutorials/tts/Vits_Training.ipynb

```diff
@@ -0,0 +1,372 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "3W8unatgYbUp"
+   },
+   "outputs": [],
+   "source": [
+    "# We can load the pre-trained model as follows\n",
+    "model = VitsModel.from_pretrained(\"tts_en_lj_vits\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "xsyBa9tIdHp4"
+   },
+   "outputs": [],
+   "source": [
+    "# VITS is a TextToWaveform\n",
+    "assert isinstance(model, TextToWaveform)\n",
+    "\n",
+    "# TextToWaveform in NeMo have two helper functions:\n",
+    "#   1. parse(self, str_input: str) which takes an English string and produces a token tensor\n",
+    "#   2. convert_text_to_waveform(self, *, tokens) which takes the token tensor and generates an audio sample\n",
+    "# Let's try it out\n",
+    "tokens = model.parse(str_input = \"Hey, this produces speech!\")\n",
+    "audio = model.convert_text_to_waveform(tokens = tokens)\n",
+    "\n",
+    "# Now we can visualize the generated spectrogram\n",
+    "# If we want to generate speech, we have to use a vocoder in conjunction to a spectrogram generator.\n",
+    "# Refer to the TTS Inference notebook on how to convert spectrograms to speech.\n",
+    "from matplotlib.pyplot import imshow\n",
+    "from matplotlib import pyplot as plt\n",
+    "import IPython.display as ipd\n",
+    "\n",
+    "ipd.Audio(audio)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zZ90eCfdrNIf"
+   },
+   "source": [
+    "# Training\n",
+    "\n",
+    "Now that we looked at the VITS model, let's see how to train a VITS Model\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "7rHG-LERrPRY"
+   },
+   "outputs": [],
+   "source": [
+    "# NeMo's training scripts are stored inside the examples/ folder. Let's grab the vits.py file\n",
+    "# as well as the vits.yaml file\n",
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/tts/vits.py\n",
+    "!(mkdir -p conf \\\n",
+    "  && cd conf \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/tts/conf/vits.yaml \\\n",
+    "  && cd ..)\n",
+    "\n",
+    "# We will also need a few extra files for handling text.\n",
+    "!(mkdir -p scripts/tts_dataset_files \\\n",
+    "  && cd scripts/tts_dataset_files \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/cmudict-0.7b_nv22.10 \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/tts_dataset_files/heteronyms-052722 \\\n",
+    "  && cd ..)\n",
+    "        \n",
+    "!(mkdir -p nemo_text_processing/text_normalization/en/data/whitelist/ \\\n",
+    "  && cd nemo_text_processing/text_normalization/en/data/whitelist/ \\\n",
+    "  && wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/nemo_text_processing/text_normalization/en/data/whitelist/lj_speech.tsv \\\n",
```

Comment:
please confirm if you need this `whitelist` file explicitly. It has been applied by default after decoupling text normalization into a new repo from NeMo.